coordinator/src/test/scala/filodb.coordinator/queryplanner/MultiPartitionPlannerSpec.scala

```diff
@@ -175,6 +175,105 @@ class MultiPartitionPlannerSpec extends AnyFunSpec with Matchers {
 
   }
 
+  it ("should not generate PromQlExec plan when partitions are local for TopLevelSubquery") {
+    val partitionLocationProvider = new PartitionLocationProvider {
+      override def getPartitions(routingKey: Map[String, String], timeRange: TimeRange): List[PartitionAssignment] =
+        List(PartitionAssignment("local", "local-url", TimeRange(timeRange.startMs, timeRange.endMs)))
+
+      override def getAuthorizedPartitions(timeRange: TimeRange): List[PartitionAssignment] =
+        List(PartitionAssignment("local", "local-url", TimeRange(timeRange.startMs, timeRange.endMs)))
+    }
+
+    val engine = new MultiPartitionPlanner(partitionLocationProvider, localPlanner, "local", dataset, queryConfig)
+    val lp = Parser.queryRangeToLogicalPlan(
+      "test{job = \"app\"}[100m:10m]",
+      TimeStepParams(endSeconds, step, endSeconds)
+    )
+
+    val promQlQueryParams = PromQlQueryParams("test{job = \"app\"}", endSeconds, step, endSeconds)
```

Comment:
This should have query "test{job = \"app\"}[100m:10m]" right?

Reply:
fixed